src/main/java/org/zalando/nakadi/security/ScopeHelper.java

```diff
@@ -0,0 +1,25 @@
+package org.zalando.nakadi.security;
+
+import org.zalando.nakadi.exceptions.IllegalScopeException;
+
+import java.util.Objects;
+import java.util.Set;
+
+public class ScopeHelper {
+
+    public static void checkScopes(final Set<String> allowedScopes, final Set<String> clientScopes) {
+        if (areScopesEmpty(allowedScopes)) {
```

Comment:
We've decided that they're can't be null, that's why you could just check allowedScopes.isEmpt()